.env.example

```diff
@@ -30,6 +33,7 @@ GITHUB_TOKEN=
 STORAGE_SECRET=
 
 # Session secret used for signing cookies in the web interface
+# Generate a new key with: $ openssl rand -hex 32
```

Comment:
LGTM - might also want to mention SESSION_SECRET in [CONTRIBUTING.md](https://github.com/integrations/slack/blob/master/CONTRIBUTING.md#configuring-a-github-app).

Reply:
Done in latest commit! Thanks!